package.json

```diff
@@ -1,17 +1,23 @@
 {
   "name": "eslint-config-mavenlint",
-  "version": "1.0.3",
+  "version": "1.1.0",
   "description": "Mavenlink ESLint config",
   "main": "index.js",
   "repository": "https://github.com/mavenlink/mavenlint.git",
   "license": "MIT",
-  "peerDependencies": {
-    "eslint": ">= 3.7.0"
+  "devDependencies": {
+    "eslint": "^3.7.0",
+    "eslint-plugin-import": "^1.13.0",
+    "eslint-plugin-jsx-a11y": "^2.1.0",
+    "eslint-plugin-react": "^6.1.2"
   },
-  "dependencies": {
-    "eslint-config-airbnb": "^10.0.1",
+  "peerDependencies": {
+    "eslint": "^3.7.0",
```

Comment:
On `npm install` with the recommendation I get the following

```
npm WARN eslint-config-airbnb@10.0.1 requires a peer of eslint@^3.3.0 but none was installed.
npm WARN eslint-config-airbnb-base@5.0.3 requires a peer of eslint@^3.3.1 but none was installed.
npm WARN eslint-plugin-import@1.16.0 requires a peer of eslint@2.x - 3.x but none was installed.
npm WARN eslint-plugin-jsx-a11y@2.2.3 requires a peer of eslint@^2.10.2 || 3.x but none was installed.
npm WARN eslint-plugin-react@6.10.3 requires a peer of eslint@^2.0.0 || ^3.0.0 but none was installed.
```

I believe there's some problems with the `package.json` on one of those packages.